src/index.md

```diff
@@ -3,7 +3,10 @@ title: Alpha design system documentation
 layout: "layouts/base.njk"
 ---
 
-# Alpha design system
-
-[English]({{ "/en/" | url }}) | [Français]({{ "/fr/" | url }})
-
+<div class="landing-page">
+  <h1>Alpha design system</h1>
+  <div class="landing-page__btns">
+    <gcds-button type="link" href="/en/">English</gcds-button>
+    <gcds-button type="link" href="/fr/" >Français</gcds-button>
```

Comment:
Just need a lang="fr" on the Français link

Reply:
Done!